scss/_patterns_subnav.scss

```
@@ -104,6 +104,7 @@
   }
 
   @include deprecate('3.0.0', 'Use .p-navigation__items, .p-navigation__item, .p-navigation__link classes instead.') {
+    // stylelint-disable-next-line selector-max-type
```

In places like this we should add also some `// FIXME` comment to note that this is not an expected place for element selector and should be fixed.

---

This is actually deprecated, so no need to fix that, maybe just add a comment to disable rule
```suggestion
    // stylelint-disable-next-line selector-max-type -- this is deprecated use, will be removed
```